org-code-codebuilderapp/app/src/main/java/org/code/codebuilderapp/AnonymousUser.java

```diff
@@ -0,0 +1,20 @@
+package org.code.codebuilderapp;
+
+import java.security.Principal;
+
+/**
+ * A principal associated with a single anonymous session.
```

Comment:
Placeholder until we have access control figured out. 
Some discussion on this here: https://codedotorg.slack.com/archives/C03CK49G9/p1613502389142700?thread_ts=1613425248.140900&cid=C03CK49G9